randovania/games/prime1/logic_database/header.json

```diff
@@ -290,25 +290,49 @@
                     "item_id": 1004
                 }
             },
-            "MainPB": {
-                "long_name": "Main Power Bomb",
+            "UnlimitedMissiles": {
+                "long_name": "Unlimited Missiles",
                 "max_capacity": 1,
                 "extra": {
-                    "item_id": 1005
+                    "item_id": 41,
```

Comment:
since this value isnt used for the custom items anymore, I'd prefer if these values would be set to 1041, 1042 etc.